src/main/java/seedu/lifeasier/tasks/Deadline.java

```diff
@@ -21,6 +21,12 @@ public Deadline(String description, LocalDateTime by, boolean isDone) {
         this.isDone = isDone;
     }
 
+    public Deadline(Task task, int editNumber) {
+        super(task, editNumber);
```

Comment:
Maybe instead of setting the editNumber through the constructor, set a method called setEditNumber() in the Task class itself instead.